annotations/src/main/java/org/p3model/annotations/ExcludeFromDocs.java

```diff
@@ -0,0 +1,4 @@
+package org.p3model.annotations;
+
+public @interface ExcludeFromDocs {
```

Comment:
This class do not have a `@Target`, and the default behavior depends of the Java version we use (and some JDK versions before 14 have [bugs](https://bugs.eclipse.org/bugs/show_bug.cgi?id=552082) related with it). Are we sure, that we do not declare explicitly the list of allowed targets?